I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 39, in script
    [male_character] "What's your name?"
Exception: Sayer b'[male_character]' is not a function or string.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 39, in script
    [male_character] "What's your name?"
  File "C:\Users\Marcus Gana\Downloads\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\Marcus Gana\Downloads\renpy-8.3.4-sdk\renpy-8.3.4-sdk\renpy\ast.py", line 598, in execute
    raise Exception("Sayer %s is not a function or string." % self.who.encode("utf-8"))
Exception: Sayer b'[male_character]' is not a function or string.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.4.24120703
Fragments 1.0
Thu Feb 20 16:19:40 2025
